css/impact.css

```diff
@@ -0,0 +1,277 @@
+/*
+===================================
+===================================
+Our Impact Styles
+===================================
+===================================
+*/
+
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+
+/* styled img container */
+
+.styled-img-container {
+	position: relative;
+}
+
+.styled-img-container img {
+	border-radius: var(--border-radius);
+	max-height: 550px;
+	object-fit: cover;
+	object-position: top;
+	aspect-ratio: 1/1;
+}
+
+.styled-img-container::before {
+	position: absolute;
+	content: " ";
+	width: 80%;
+	aspect-ratio: 1/1;
+	z-index: -1;
+	background-color: var(--color-amaranth);
+	border-radius: var(--border-radius);
+}
+
+.styled-img-container.style-left::before {
+	transform: translate(-70px, -65px);
+}
+
+.styled-img-container.style-right::before {
+	transform: translate(200px, -65px);
+}
+/* 
+===================================
+Hero 
+===================================
+*/
+
+/* background and layout adjustments  */
+.hero {
+	background: linear-gradient(
+			107deg,
+			rgba(9, 9, 9, 0.8) 9.51%,
+			rgba(9, 9, 9, 0) 76.72%
+		),
+		url("../assets/images/pages/impact/hero-impact.png") top/cover;
+	display: flex;
+	align-items: center;
+}
+
+/* about heading layout and sizing */
+.hero-text-container {
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Technology & Stewardship Cards
+===================================
+*/
+.section-2-col .wrapper {
+	display: flex;
+	gap: var(--spacing-xl);
+	padding: var(--spacing-xl);
+
+	width: var(--wrapper-width);
+	max-width: var(--wrapper-max-width);
+	margin: 0 auto;
+}
+.section-2-col .styled-img-container,
+.section-2-col .text-container {
+	width: 50%;
+}
+
+.section-2-col .text-container {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* 
+===================================
+Programs Gallery
+===================================
+*/
+.programs {
+	position: relative;
+}
+
+/* overlapping background */
+
+.programs::before {
+	position: absolute;
+	content: "";
+	z-index: -1;
+	top: 215px;
+
+	width: 100%;
+	height: 60%;
+	background: linear-gradient(
+		180deg,
+		#d3c53f 0%,
+		#d3c53f 26.04%,
+		#92a32d 58.85%,
+		#b37122 100%
+	);
+}
+
+/* layout */
+.program-gallery {
```

Comment:
This needs a border-radius

<img width="267" alt="Screen Shot 2024-01-10 at 12 03 20 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/44c05926-525a-4229-a052-19ff6fdae361">

Reply:
added in!